src/qt/forms/debugwindow.ui

```diff
@@ -1395,6 +1395,19 @@
                  </property>
                 </widget>
                </item>
+               <item row="20" column="0">
+                <spacer name="verticalSpacer_3">
+                 <property name="orientation">
+                  <enum>Qt::Vertical</enum>
+                 </property>
+                 <property name="sizeHint" stdset="0">
+                  <size>
+                   <width>20</width>
+                   <height>10</height>
```

Comment:
no real objection, but upstream is 40, why the change?

Reply:
see https://github.com/dashpay/dash/pull/4543#discussion_r735677143: I compiled it and it behaved weird (imo) with 40 - there is enough free space and yet you see scrollbars

Reply:
Also, I have no idea why `UdjinM6 dismissed PastaPastaPasta’s stale review via 8179bd5` happened 🤷‍♂️  pls re-review @PastaPastaPasta